README.md

```diff
@@ -49,3 +49,12 @@ Here are some web pages that you may need to refer to:
       -https://www.kaggle.com/tathor/yugioh-trading-cards-dataset/data
       - YuGiOh is a card game that I have a lot of interest in and I think it would be fun to explore the most common card type or              monster attributes given the over 6,000 cards included in this data set.
   
+
+Cody Makinson
+1. I want to analyze NHL data from the birth of the NHL original 6 hockey teams.
+2. I really enjoyed pokemon growing up, so I would use the data we previously used in an assignment. I want to see if there is a type bias, among other statitical analysis.
+3.Weather data. It is cyclical and a different kind of data than the other two.
+
+https://www.justsportsstats.com/hockeyindex.php?league=NHL *I just need to download in excel* 
```

Comment:
I wasn't able to see how to download any of these as files. You should prove to yourself that they are available and take a look at the files you are thinking about using in Excel to get a feel for the data.